src/com/mishiranu/dashchan/ui/posting/dialog/ReencodingDialog.java

```diff
@@ -61,7 +68,10 @@ public Dialog onCreateDialog(Bundle savedInstanceState) {
 			radioButton.setId(IDS[i]);
 			radioGroup.addView(radioButton);
 		}
-		radioGroup.check(IDS[0]);
+		radioGroup.check(IDS[JPEG_POSITION]);
+		LinearLayout jpegToJpgLayout = new LinearLayout(context);
```

Comment:
I think frame layout is better here because its faster than linear layout and we don't need linear layout's capabilities here.
```suggestion
		FrameLayout jpegToJpgLayout = new FrameLayout(context);
```